app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -52,7 +52,7 @@ def self.raw_connect(server, port, username, password, service = "Service")
     Ovirt.const_get(service).new(params)
   end
 
-  def connect(options = {})
+  def other_connect(options = {})
```

Comment:
I feel like the "single `connect` calls `raw_connect`" pattern is more a consequence of how the lower level libraries are structured, and maybe not an API we should be replicating at all.

cc @blomquisg